docs/UserGuide.md

```diff
@@ -719,8 +720,18 @@ Edit INDEX? (yes/no)
    * If yes: the specified session will be updated with the specified details.
    * If no: the specified session will not be updated.
 
-Examples:
-* `editses 2 s/Tutorial1 d/10/10/2020` followed by `yes` edits the 2nd session in the session list with a new session name `Tutorial1` and a new session date `10/10/2020`.
+</div>
+
+:alert-warning: You will be exited from any currently entered session upon using the command.
+
+:green_book: Examples:
+* `editses 1 s/Tutorial 6 d/10/10/2020` followed by `yes` edits the 1st session in the session list with a new session name `Tutorial 6` and a new session date `10/10/2020`.
```

Comment:
Perhaps these parts could be wrapped in their respective div boxes for better organisation of info? Noticed the same issue in other areas as well.

Reply:
Edited with thanks